### ISSUE TITLE
Issues in student_admissions

### ISSUE THREAD

**Issue by WikiGenius** [udacity/deep-learning#290]
1_ wrong calculation: error_term_formula(y, output)
It is supposed to return (y-y_hat),
but in solution (y-y_hat) * sigmoid_prim(h) I don't know why and we don't use any hidden layer,
we have the number of weights = number of features.
and del E / del W_j = -(y-y_hat)* x_j

2_ We don't use bias in our calculations, so it should be column bias = 1 in processing data

3_ Why in epochs we use mean square error rather than cross-entropy?

4_ I know one-hot encoding is used for categorical variables but we used it in rank and there is order so we lose the order.

5_ 
(1)derivative the whole cross-entropy: -1/m sum( (y-y_hat) * x)
and here we use del_w to integrate the values 
,and
(2)the derivative the cross-entropy at each data point is (y-y_hat) * x
So, which one is better in performance.

**Comment by WikiGenius** [comment]
1_ Here, we don't use the error log loss function, we use mean square error.
2_ I think, it has a small impact on the data here on one layer NN, so it is useless.
3_ Because, we use mean square error in backpropagation.
4_ The order not only in place but as value, so if order2 is not twice the value order 1 and order 3  is not 3 times order1.
So, it is negative information to train the data.
5_ There is no problem, use these options, and the accuracy of test data will tell you what the best.

**Comment by abhiojha8** [comment]
We suggest you refer to the Knowledge hub and reach out to your assigned mentor for user-specific issues. Thanks!
